articles/synapse-analytics/overview-faq.yml

```diff
@@ -101,7 +106,17 @@ sections:
         answer: |
           At this time, you must manually recreate your Azure Data Factory pipelines and related artifacts by exporting the JSON from the original pipeline and importing it into your Synapse workspace.
 
-  - name: Apache Spark
+      - question: |
+          How do I use an Apache Spark Job Definition?
+        answer: |
+          Check out this [Quick Start Guide](https://docs.microsoft.com/en-us/azure/synapse-analytics/quickstart-transform-data-using-spark-job-definition)
+
+      - question: |
+          Can I call notebooks from ADF pipelines?
+        answer: |
+          There are two options for this use case. One option is to keep pipelines in ADF, and you will have to wrap in a web activity. The other option is to migrate the pipelines to Synapse.
```

Comment:
Could you please add links to each of these options?
Doc for differences between ADF and synapse pipelines:
https://docs.microsoft.com/en-us/azure/synapse-analytics/data-integration/concepts-data-factory-differences

Doc for web activity in ADF:
https://docs.microsoft.com/en-us/azure/data-factory/control-flow-web-activity

Tool for migration:
https://github.com/Azure-Samples/Synapse/tree/main/Pipelines/ImportADFtoSynapse

Reply:
Added in commit cb70304